config/routes.yaml

```diff
@@ -63,3 +63,23 @@ bulkCreateUsers:
   path: '%app.route_prefix%/v1/bulk-create-users'
   methods: POST
   defaults: { _controller: OAT\SimpleRoster\Action\CreateEntity\BulkCreateUsersAction }
+
+listLtiInstances:
+  path: '%app.route_prefix%/v1/lti-instances'
+  methods: GET
+  defaults: { _controller: OAT\SimpleRoster\Action\LtiInstance\ListAction }
+
+createLtiInstances:
+  path: '%app.route_prefix%/v1/lti-instance'
```

Comment:
here and below the rule `plural nouns` also should be applied

Reply:
fixed